src/coredump/coredump-ratelimit.c

```diff
@@ -0,0 +1,170 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <sys/statvfs.h>
```

Comment:
style nit: we usually separate system and local headers by an empty line, and sort the two blocks alphabetically